utils/dtoverlay/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dtoverlay
+PKG_VERSION:=2016-07-06
+PKG_RELEASE=$(PKG_SOURCE_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/ya-mouse/dtoverlay.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=d503ddae11d62f12762b2fd3a52d37da55645d4b
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
+
+CMAKE_INSTALL:=1
+
+PKG_LICENSE:=PublicDomain
```

Comment:
Seems to be wrong! When I look at C files etc, this should be something like BSD-3-Clause and/or GPL. Please review.